apps/meteor/ee/app/license/server/license.ts

```diff
@@ -237,11 +220,11 @@ class LicenseClass {
 	}
 
 	async canEnableApp(source: LicenseAppSources): Promise<boolean> {
-		if (!this.Apps?.isInitialized()) {
+		if (!Apps.isInitialized()) {
 			return false;
 		}
 
-		return isUnderAppLimits({ appManager: this.Apps.getManager() as AppManager }, this.appsConfig, source);
+		return isUnderAppLimits(this.appsConfig, source);
 	}
```

Comment:
this is not supposed to be a license service concern

Reply:
In the same vein, you would say that the `canAddNewUser` method is not supposed to be a license service concern as well, correct?

Reply:
sure. the best explanation I can give you: I was not involved on `canAddNewUser` review  🤷